module/exporter/exporter.py

```diff
@@ -0,0 +1,8 @@
+import json
+
+
+def export_to_json(all_statistic, country_statistic):
+    with open('result.json', 'w') as fp:
```

Comment:
1. Put a flexible filename
2. Return something to get it checked from run.py ( Files cannot be written in some cases )